scripts/rpc-api-generation/index.ts

```diff
@@ -22,6 +22,14 @@ type ProtoServiceDefinition = {
   >
 }
 
+const deprecatedApiMethods: Record<string, string[]> = {
```

Comment:
wait, why are we duplicating all the depecreated methods here ?
we already have this info on the .proto file. We can do some regex to find all the methods that are depcreated.
This way we have only one source of truth